bin/docker/docker-compose.nginx.yml

```diff
@@ -17,10 +17,10 @@ services:
     image: nginx:1.13-alpine
     container_name: mainflux-nginx
     volumes:
-      - $PWD/docker/nginx.conf:/etc/nginx/nginx.conf
-      - $PWD/docker/ssl/certs/mainflux-server.crt:/etc/ssl/certs/mainflux-server.crt
-      - $PWD/docker/ssl/certs/mainflux-server.key:/etc/ssl/private/mainflux-server.key
-      - $PWD/docker/ssl/dhparam.pem:/etc/ssl/certs/dhparam.pem
```

Comment:
@Essarelle looks like you are not launching the script from the correct dir, as noted here: https://github.com/mainflux/mainflux#docker

You should `cd mainflux/bin` and launch from there (i.e. from the same dir where `mainflux-docker.sh` lives).